Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -925,6 +926,52 @@ void HoI4::World::reportIndustryLevels() const
 }
 
 
+void HoI4::World::setTrainMultipliers()
```

Comment:
This is the weirdest loop ever. Why not make a set of numbers, iterate over both chains and when one is done break away.

That would incidentally have the bonuses plainly visible at the start of the function.

Reply:
Don't code while exhausted, tipsy, or both. That's a much better solution.